bucket/btop-lhm.json

```
@@ -18,7 +18,7 @@
         "Get-ChildItem \"$dir\\btop4win.exe\" | Rename-Item -NewName { $_.name -Replace '4win', '' }",
         "if (!(Test-Path \"$persist_dir\\btop.conf\")) { New-Item \"$dir\\btop.conf\" | Out-Null }"
     ],
-    "pre_uninstall": "sc stop R0btop | Out-Null",
+    "pre_uninstall": "sc.exe stop R0btop | Out-Null",
```

Add admin rights check to match suggestion in https://github.com/ScoopInstaller/Main/pull/3897#issuecomment-1242946583
```suggestion
    "pre_uninstall": [
        "if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
        "sc.exe stop R0btop | Out-Null"
    ],
```